test/extended/builds/pipeline_origin_bld_rhelimagesonly.go

```diff
@@ -150,213 +150,216 @@ var _ = g.Describe("[sig-devex][Feature:JenkinsRHELImagesOnly][Slow] openshift p
 
 	g.Context("Sync plugin tests", func() {
 
-		g.It("using the ephemeral template", func() {
-			defer cleanup("jenkins-ephemeral")
-			setupJenkins("jenkins-ephemeral")
-
-			//TODO - for these config map slave tests, as well and the imagestream/imagestreamtag
-			// tests ... rather than actually running the pipelines, we could just inspect the config in jenkins
-			// to make sure the k8s pod templates are there.
-			// In general, while we want at least one verification somewhere in pipeline.go that the agent
-			// images work, we should minimize the actually running of pipelines using them to only one
-			// for each maven/nodejs
-			g.By("Pipeline using nodejs agent and client plugin")
-
-			g.By("should build and complete successfully", func() {
-				g.By("create pipeline strategy build using nodejs agent and client plugin")
-				err := oc.Run("create").Args("-f", "https://raw.githubusercontent.com/openshift/origin/master/examples/jenkins/pipeline/nodejs-sample-pipeline.yaml").Execute()
-				o.Expect(err).NotTo(o.HaveOccurred())
-
-				g.By("starting the pipeline build and waiting for it to complete")
-				// this just does sh "mvn --version"
-				br, err := exutil.StartBuildAndWait(oc, "nodejs-sample-pipeline")
-				if err != nil || !br.BuildSuccess {
-					debugAnyJenkinsFailure(br, oc.Namespace()+"-nodejs-sample-pipeline", oc, true)
-					exutil.DumpBuilds(oc)
-				}
-				br.AssertSuccess()
-
-				g.By("getting job log, make sure has success message")
-				out, err := j.GetJobConsoleLogsAndMatchViaBuildResult(br, "Finished: SUCCESS")
-				o.Expect(err).NotTo(o.HaveOccurred())
-				g.By("making sure job log ran with our nodejs pod template")
-				o.Expect(out).To(o.ContainSubstring("Running on nodejs"))
+		g.Describe("Sync plugin tests", func() {
```

Comment:
nit - stutters with the context above.

Reply:
will address momentarily

Reply:
though I'll make Context be the empty string for consistency